docs/index.md

```diff
@@ -154,27 +154,37 @@ Paste this code into terminal and restart your IDE:
 For Linux/MacOs:
 
 ```bash
-ln -s ../node_modules/kakunin/src/step_definitions/elements.ts kakunin-elements.ts
-ln -s ../node_modules/kakunin/src/step_definitions/debug.ts kakunin-debug.ts
-ln -s ../node_modules/kakunin/src/step_definitions/file.ts kakunin-file.ts
-ln -s ../node_modules/kakunin/src/step_definitions/form.ts kakunin-form.ts
-ln -s ../node_modules/kakunin/src/step_definitions/email.ts kakunin-email.ts
-ln -s ../node_modules/kakunin/src/step_definitions/generators.ts kakunin-generators.ts
-ln -s ../node_modules/kakunin/src/step_definitions/navigation.ts kakunin-navigation.ts 
-ln -s ../node_modules/kakunin/src/step_definitions/performance.ts kakunin-performance.ts 
+ln -s ../node_modules/kakunin/dist/web/step_definitions/elements.js web-kakunin-elements.js
+ln -s ../node_modules/kakunin/dist/web/step_definitions/navigation.js web-kakunin-navigation.js 
+ln -s ../node_modules/kakunin/dist/web/step_definitions/form.js web-kakunin-form.js
+ln -s ../node_modules/kakunin/dist/web/step_definitions/tabs.js web-kakunin-tabs.js
+ln -s ../node_modules/kakunin/dist/web/step_definitions/debug.js web-kakunin-debug.js
+ln -s ../node_modules/kakunin/dist/web/step_definitions/performance.js web-kakunin-performance.js 
+ln -s ../node_modules/kakunin/dist/web/step_definitions/screenshots.js web-kakunin-screenshots.js 
+
```

Comment:
do we need those empty spaces?

Reply:
I added this to separate sections which are going to different folders, but now if we won't split all of them on folders I can remove it